pfs_middleware/pfs_middleware/middleware.py

```diff
@@ -778,6 +778,12 @@ def __init__(self, app, conf, logger=None):
     def __call__(self, req):
         vrs, acc, con, obj = utils.parse_path(req.path)
 
+        # Alternate way to specify bypass mode: /proxyfs/AUTH_acct/...
+        if vrs == 'proxyfs':
+            req.headers['X-Bypass-ProxyFS'] = 'true'
```

Comment:
Should we continue to support using a header for bypass mode at all? As things stand, the upgrade path looks hairy -- old code that _only_ knew about the header will start getting a bunch of 404s on upgrade now that we've stopped clearing memcache.

If we drop the header, we could switch this all to be something more like

``` python
is_bypass_request = False
if vrs == 'proxyfs' and self.bypass_mode in ('read-only', 'read-write'):
    req.path_info = req.path_info.replace('/proxyfs/', '/v1/', 1)
    vrs = 'v1'
    is_bypass_request = True
```